README.md

```diff
@@ -42,5 +43,6 @@ _Note: Be sure to replace `<BRICK_NAME>` with one of the bricks mentioned_
 [model_json]: https://github.com/gianfrancopapa/bricks/tree/main/model_json
 [rest_frog]: https://brickhub.dev/bricks/rest_frog
 [ui_package]: https://brickhub.dev/bricks/ui_package
+[somnio_starter]: https://brickhub.dev/bricks/somnio_starter
```

Comment:
somnio_starter is not published to brickhub yet :) so we should change this to the github url